sbt-scalajs-bundler/src/main/scala/scalajsbundler/ExternalCommand.scala

```diff
@@ -44,9 +44,9 @@ object ExternalCommand {
     */
   def install(installDir: File, useYarn: Boolean, logger: Logger)(npmPackages: String*): Unit =
     if (useYarn) {
-      Yarn.run("add" +: "--non-interactive" +: npmPackages: _*)(installDir, logger)
+      Yarn.run("add" +: "--mutex" +: "file:/tmp/.yarn-mutex" +: "--non-interactive" +: npmPackages: _*)(installDir, logger)
```

Comment:
I guess this is not cross platform?

We should probably get passed the `yarnMutexFile` and generate it from sbt.

Reply:
Good point!

Reply:
Or just use `Yarn.synchronized { ... }` around that piece of code? Only works for multiple calls within the same sbt process, though.

Reply:
Maybe it's simpler to use the network-mutex: `--mutex network`. Will this work on windows?